migrations/20151117173053_add_datetime_columns.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
```

Comment:
I think the timestamp might be better because knex generates a migration template for you with one command and it timestamps it based on this format. I feel like for consistency, following whatever knex does would be easier